snapshotTree copies the git index with a plain copy that gives it a fresh mtime

--- a/src/utils/git.js
+++ b/src/utils/git.js
@@ -64,10 +64,7 @@
     const indexFile = join(gitDir, 'index')
     const indexCopyFile = join(scratchDir, 'index')
 
-    const { atime, mtime } = statSync(indexFile)
-
     copyFileSync(indexFile, indexCopyFile)
-    utimesSync(indexCopyFile, atime, mtime)
 
     await git(repoDir, 'add -u', { env: { GIT_INDEX_FILE: indexCopyFile } })
 
